Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM registry.access.redhat.com/ubi8/ubi
+
+COPY dist/ec_linux_amd64 /usr/bin/ec
```

Comment:
NIT: There's a symlink `dist/ec` that points to the current architecture. Maybe use that instead?

Reply:
The build failed when I tried that. Maybe it doesn't follow symlinks?